android/build.gradle

```diff
@@ -143,14 +143,15 @@ repositories {
 
 def media3_version = safeExtGet('media3Version')
 def kotlin_version = safeExtGet('kotlinVersion')
+def androidxCode_version = safeExtGet('androidxCore')
```

Comment:
```suggestion
def androidxCore_version = safeExtGet('androidxCore')
```
Isn't that supposed to be androidxCore_version ?

Reply:
You are right, let's keep homogeneity ! Thank you